### ISSUE TITLE
Disco Diffusion  v4.1 is available

### ISSUE THREAD

**Issue by EddyChen0309** [tnwei/vqgan-clip-app#14]
Hi tnwei，

Did you notice that disco diffusion now is available and the  effect of generating images is further improved 

Disco Diffusion v4.1 - Now with Video Inits, Recovery, DDIM Sharpen and improved UI
https://github.com/Midgraph/disco-diffusion

google colab
https://colab.research.google.com/drive/1sHfRn5Y0YKYKi1k-ifUSBFRNJ8_1sa39

Your app is awesome, hope this feature can be integrated in the future.

Eddy,thanks for your understanding!

**Comment by tnwei** [comment]
Yeah! It's awesome isn't it. I do hope to incorporate it one day, let's see if I can find time for it

**Comment by illtellyoulater** [comment]
Hi guys, could you please explain what are the main differences between this project and Disco Diffusion?
